### DISCUSSION TITLE
Extend session length before logout

### DISCUSSION THREAD

**Discussion by lonix1** [adminerevo/adminerevo#82] [Q&A]
I often get logged out of adminer.

Is there a config option to extend the session length?

**Comment by nitodico** [comment]
Weird. Check your my.ini looks more like a Mysql issue, check if happen
with CLI version what else have you tried

**Reply by lonix1** [reply]
I hoped there was a config file I could use to set the timeout, but there isn't.

 There are `adminer.php` and `index.php`, but I'm not a php dev, so I wouldn't know how to change them - also, I'm running adminer in a docker container, so I couldn't do it that way anyway. I was expecting an editable "config file".

**Comment by nitodico** [comment]
Check if this helps not sure if its and adminerevo issue but more like a
mysql problem

https://stackoverflow.com/questions/14726789/how-can-i-change-the-default-mysql-connection-timeout-when-connecting-through-py

**Comment by hjsleijster** [comment]
Being logged out of Adminer doesn't have anything to do with a mysql connection timeout. A connection with mysql is only made when executing a query.

I'm getting logged out of Adminer very randomly. The php.ini file says a session should be at least 24 minutes. This works for all websites except for adminer.
